src/HomePage/Heading/Heading.js

```diff
@@ -3,8 +3,11 @@ import "./Heading.scss";
 
 const Heading = () => {
   return (
-    <div>
-      <h1 className="heading">Hello World, I'm Anita</h1>
+    <div className="header">
+      <h1 className="heading">
+        Hello World, I'm Anita <br />
```

Comment:
Using br's are frowned upon generally. Could you move this into a seperate block element?

Reply:
Have changed